meinberlin/apps/dashboard/templatetags/meinberlin_dashboard_tags.py

```diff
@@ -35,7 +35,7 @@ def has_unpublishable_modules(project):
     another module published for (added to) the project.
     """
     return (project.is_draft
-            and project.published_modules.count() <= 1)
+            and project.published_modules.count() > 1)
```

Comment:
I wonder if we should also allow it when `== 0` - e.g. when creating a new project and then having several unpublished modules. Anyway, that's for later.